src/custom/pages/App/index.tsx

```diff
@@ -18,12 +18,12 @@ import { Loading } from 'components/FlashingLoading'
 
 // Sync routes
 import Account from 'pages/Account'
+import Swap from 'pages/Swap'
```

Comment:
Why not using lazy-loading here?

Reply:
It's a part of optimization.
Delay before swap widget rendering:
With LL: 1.5-3 sec
Synchronously: ~100ms delay

The swap chunk size is only 107kb gzip.

Reply:
Ok then :+1: